src/main/java/qna/domain/Answers.java

```diff
@@ -1,23 +1,33 @@
 package qna.domain;
 
+import java.util.ArrayList;
 import java.util.List;
 import java.util.Objects;
 
+import javax.persistence.Embeddable;
+import javax.persistence.OneToMany;
+
 import qna.CannotDeleteException;
 
+@Embeddable
 public class Answers {
-	private final List<Answer> answers;
+	@OneToMany(mappedBy = "question")
+	private List<Answer> answers;
 
 	public Answers(List<Answer> answers) {
 		this.answers = answers;
 	}
 
+	protected Answers() {
+		this.answers = new ArrayList<>();
+	}
+
 	public boolean haveNotOwner(User loginUser) {
 		return answers.stream()
 			.anyMatch(answer -> !answer.isOwner(loginUser));
 	}
 
-	public List<Answer> getAnswers() {
+	public List<Answer> get() {
 		return answers;
 	}
```

Comment:
일급 콜렉션에서 내부 콜렉션을 get으로 어쩔 수 없이 내보낼 때는 불변 리스트로 바꿔 넘겨주시는 편이 좋습니다.
```suggestion
	public List<Answer> get() {
		return Collections.unmodifiableList(answers);
	}
```
요런 방법도 가능하니 참고 부탁드릴께요!